server/model/events/events.go

```diff
@@ -158,51 +158,6 @@ func TriggergRPCUnreachableHostError(testID id.ID, runID int, err error) model.T
 	}
 }
 
-func TraceFetchingStart(testID id.ID, runID int) model.TestRunEvent {
-	return model.TestRunEvent{
-		TestID:              testID,
-		RunID:               runID,
-		Stage:               model.StageTrace,
-		Type:                "FETCHING_START",
-		Title:               "Starting the trace fetching process",
-		Description:         "Starting the trace fetching process",
-		CreatedAt:           time.Now(),
-		DataStoreConnection: model.ConnectionResult{},
-		Polling:             model.PollingInfo{},
-		Outputs:             []model.OutputInfo{},
-	}
-}
-
-func TraceQueuedInfo(testID id.ID, runID int) model.TestRunEvent {
-	return model.TestRunEvent{
-		TestID:              testID,
-		RunID:               runID,
-		Stage:               model.StageTrace,
-		Type:                "QUEUED_INFO",
-		Title:               "Trace Run has been queued to start the fetching process",
-		Description:         "Trace Run has been queued to start the fetching process",
-		CreatedAt:           time.Now(),
-		DataStoreConnection: model.ConnectionResult{},
-		Polling:             model.PollingInfo{},
-		Outputs:             []model.OutputInfo{},
-	}
-}
```

Comment:
This event can be captured by the `TracePollingStart`, because they are the same. Thinking about that, I've removed this event and focused on send the `TracePollingStart` event.